src/LogHandler.php

```diff
@@ -17,10 +17,10 @@ class LogHandler extends AbstractProcessingHandler
 
     /**
      * @param \Honeybadger\Contracts\Reporter $honeybadger
-     * @param int $level
+     * @param $level
      * @param bool $bubble
      */
-    public function __construct(Reporter $honeybadger, int $level = Logger::DEBUG, bool $bubble = true)
+    public function __construct(Reporter $honeybadger, $level = Logger::ERROR, bool $bubble = true)
```

Comment:
Removed the `int` because this can also be a string level ('error') or (in monolog v3) an enum. Also raised the default level to error.